action-server/alembic/versions/00002_test_profiles.py

```diff
@@ -0,0 +1,233 @@
+    )
+)
+
+TEST_PROFILE_3 = (
+    TestProfile(3)
+    .set_reminder_attributes(
+        first_name="Maurice",
+        phone_number="15145553456",
+        language="fr",
+        province="ca-nt",
+        age_over_65=True,
+        preconditions=False,
+        has_dialogue=False,
+    )
+    .set_assessment_attributes(
+        symptoms="moderate", has_fever=False, has_cough=False, has_diff_breathing=True
+    )
+)
+
+TEST_PROFILE_4 = (
+    TestProfile(4)
+    .set_reminder_attributes(
+        first_name="Jane",
+        phone_number="15145554567",
+        language="fr",
+        province="ca-qc",
+        age_over_65=True,
+        preconditions=True,
+        has_dialogue=True,
+    )
+    .set_assessment_attributes(
+        symptoms="moderate", has_fever=False, has_cough=False, has_diff_breathing=True
+    )
+)
+
+TEST_PROFILES = [TEST_PROFILE_1, TEST_PROFILE_2, TEST_PROFILE_3, TEST_PROFILE_4]
+
+
+def upgrade():
+    _clear_reminder_table()
+    _clear_assessment_table()
+
+    reminder_table = _create_reminder_table_construct()
+    assessment_table = _create_assessment_table_construct()
+
+    _create_test_profiles(reminder_table, assessment_table, TEST_PROFILES)
+
+
+def downgrade():
+    _delete_test_profiles(TEST_PROFILES)
+
+
+def _clear_reminder_table():
+    op.execute(f"DELETE FROM {REMINDER_TABLE}")
+
+
+def _clear_assessment_table():
+    op.execute(f"DELETE FROM {ASSESSMENT_TABLE}")
+
+
+def _create_reminder_table_construct():
+    log.info("Creating reminder table construct.")
+
+    return table(
+        REMINDER_TABLE,
+        Column("id", Integer, primary_key=True),
+        Column(
+            "created_at",
+            DateTime(timezone=True),
+            server_default=func.current_timestamp(),
+            nullable=False,
+        ),
+        Column("last_reminded_at", DateTime(timezone=True),),
+        Column("timezone", String(), CheckConstraint("is_timezone(timezone)"),),
+        Column("is_canceled", Boolean(), nullable=False, server_default="false"),
+        Column("attributes", JSONB, nullable=False),
+    )
+
+
+def _create_assessment_table_construct():
+    log.info("Creating assessment table construct.")
+
+    return table(
+        ASSESSMENT_TABLE,
+        Column("id", Integer, primary_key=True),
+        Column(
+            "reminder_id",
+            Integer,
+            ForeignKey("reminder.id", onupdate="CASCADE", ondelete="CASCADE"),
+        ),
+        Column(
+            "completed_at",
+            DateTime(timezone=True),
+            server_default=func.current_timestamp(),
+            nullable=False,
+        ),
+        Column("attributes", JSONB, nullable=False),
+    )
+
+
+def _create_test_profiles(reminder_table, assessment_table, profiles):
+    log.info("Adding test profiles to reminder table.")
+
+    reminders = [test_profile.create_reminder() for test_profile in profiles]
+
+    op.bulk_insert(reminder_table, reminders)
```

Comment:
Autre trucs: Après création des reminders, il faudrait utiliser `hashids` et afficher l'ID hashé des comptes de profil dans les logs. Sans ça, on ne pourra pas faire le daily check-in (puisque ça prend le hashid dans l'URL)